contracts/Transfer.sol

```diff
@@ -23,17 +23,17 @@ contract Transfer {
      * @notice processes the state transition of a commitment
      * */
     function processTransferCommit(
-        bytes32 stateRoot,
+        bytes32 postStateRoot,
+        bytes32 previousStateRoot,
```

Comment:
I'd call this param `currentStateRoot` or `stateRoot` here to better reflect the meaning of the variable during the function execution.